<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>standard_params</class>
 <widget class="QWidget" name="standard_params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="target_label">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="target_field"/>
     </item>
     <item>
      <widget class="QPushButton" name="target_button">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/open-iconic-master/png/3x/target-3x.png</normaloff>:/icons/icons/open-iconic-master/png/3x/target-3x.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="uniq_checkbox">
       <property name="text">
        <string>Unique results (per target)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
